### ISSUE TITLE
Would be good to have an explicit Security/Privacy section

### ISSUE THREAD

**Issue by g-ortuno** [explainers-by-googlers/template#2]
During review a lot of security related questions came up which were already answered in the explainer. Security also requested a Security and Privacy considerations section. Would it make sense to add it for clarity?

**Comment by jyasskin** [comment]
Yes, I'll write up a PR, or you're welcome to if you get to it first.
